minds/minds.py

```diff
@@ -100,9 +99,20 @@ def update(
             f'/projects/{self.project}/minds/{self.name}',
             data=data
         )
+
         if name is not None and name != self.name:
             self.name = name
 
+        refreshed_mind = self.client.minds.get(self.name)
+        self.model_name = refreshed_mind.model_name
+        self.provider = refreshed_mind.provider
+        self.prompt_template = refreshed_mind.prompt_template
+        self.parameters = refreshed_mind.parameters
+        self.created_at = refreshed_mind.created_at
+        self.updated_at = refreshed_mind.updated_at
+        self.datasources = refreshed_mind.datasources
```

Comment:
Maybe make it trickier? 
```python
mind = self.client.minds.get(self.name)
self.__dict__ = mind.__dict__.copy()
```

Reply:
We can do it. But can we keep it simple to read and understand?